src/features/island/common/chest-reward/ChestReward.tsx

```diff
@@ -37,10 +49,11 @@ export const ChestReward: React.FC<Props> = ({
 }) => {
   const { gameService } = useContext(Context);
   const isNew = useSelector(gameService, isNewGame);
```

Comment:
I would consider extending this logic. Currently captcha does not show for new players (neither the chest puzzle or goblin puzzle).

In this case here I feel it would be better if there was no challenge at all.

This is how we currently bypass showing a captcha (checking the `isNew`)

```
  useEffect(() => {
    if (reward && !isNew) {
      setLoading(true);
      setTimeout(() => setLoading(false), 500);
    }
  }, [reward]);
```